src/main/java/io/cryostat/net/web/http/api/v2/graph/StartRecordingOnTargetMutator.java

```diff
@@ -133,13 +133,17 @@ public HyperlinkedSerializableRecordingDescriptor getAuthenticated(
                 cd,
                 conn -> {
                     boolean restart = false;
+                    String replace = "never";
                     RecordingOptionsBuilder builder =
                             recordingOptionsBuilderFactory
                                     .create(conn.getService())
                                     .name((String) settings.get("name"));
                     if (settings.containsKey("restart")) {
                         restart = Boolean.TRUE.equals(settings.get("restart"));
                     }
+                    if (settings.containsKey("replace")) {
```

Comment:
One point on the graphql changes, since `replace` is a new field within the GraphQL `doStartRecording` mutator schema, we need to register the new field in the schema which is located in the file here: https://github.com/cryostatio/cryostat/blob/main/src/main/resources/types.graphqls#L168

it will probably look something like:

```graphql
input RecordingSettings {
    restart: Boolean
    replace: String
   ...omitted
   ...omitted
    metadata: Object
}
```
Then you can test the query manually with something like 

```graphql
query { 
    targetNodes(filter: { annotations: "PORT == 0" }) {
        doStartRecording(recording: { 
            name: "grap-itest", 
            duration: 30,
            template: "Profiling", 
            templateType: "TARGET",
            restart: true,
            replace: "never"
        })
        { 
            name state duration 
        }
    } 
}
```

More info about Java and graphql here: https://www.graphql-java.com/documentation/data-mapping